eth/handler.go

```diff
@@ -439,8 +442,21 @@ func (h *handler) BroadcastBlock(block *types.Block, propagate bool) {
 			log.Error("Propagating dangling block", "number", block.Number(), "hash", hash)
 			return
 		}
-		// Send the block to a subset of our peers
-		transfer := peers[:int(math.Sqrt(float64(len(peers))))]
```

Comment:
Why not just something like this?
```go
transfer := peers[:max(len(peers), min(8, int(math.Sqrt(float64(len(peers))))))]
```

Reply:
Here we not only want to broadcast to more nodes, but also want to broadcast to as different nodes as possible each time. Because although the peers field of peerSet is a map structure, the traversal is random, but the peersWithoutBlock function returns a slice, so I hope that the nodes selected from the slice will also have a certain randomness